packages/dashboard/src/components/iot-dashboard/iot-dashboard.tsx

```diff
@@ -307,6 +317,14 @@ export class IotDashboard {
     this.onGestureEnd();
   }
 
+  @Listen('keydown')
+  onKeyDown({ key }: KeyboardEvent) {
```

Comment:
Is there a button to do this also? I know figjam doesn't have this, but they have a button that shows on hover to duplicate. I think the keyboard shortcuts are great for people doing a lot of dashboard work, but a first time user might not think to do this.

Reply:
I'm unsure. I agree on the discoverability. As we start fleshing out the UX of the dashboard more we can explore that option

Reply:
Ya, doesn't change the underlying method. Just a thought to build in! lgtm otherwise.